operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/util/BulkDataExportUtil.java

```diff
@@ -39,11 +40,19 @@
 public class BulkDataExportUtil {
 
     private static JobIdEncodingTransformer transformer = new JobIdEncodingTransformer();
+    private static Set<String> RESOURCE_TYPES = ModelSupport.getResourceTypes(false).stream()
+                                                    .map(m -> m.getSimpleName())
+                                                    .filter(f -> !isFiltered(f))
```

Comment:
we shouldn't need this line because `ModelSupport.getResourceTypes(false)` should only return concrete resource types
```suggestion
```